Plugins/Flow.Launcher.Plugin.Explorer/ContextMenu.cs

```diff
@@ -450,7 +450,8 @@ private Result CreateOpenWithMenu(SearchResult record)
                     Process.Start("rundll32.exe", $"{Path.Combine(Environment.SystemDirectory, "shell32.dll")},OpenAs_RunDLL {record.FullPath}");
                     return true;
                 },
-                IcoPath = Constants.ShowContextMenuImagePath
+                IcoPath = Constants.ShowContextMenuImagePath,
```

Comment:
Is there also a more fitting image for this?

Reply:
Generally @onesounds is providing advice for this, but since he is really busy now we can just use that.